api/server/router/swarm/cluster_routes.go

```diff
@@ -265,6 +273,29 @@ func (sr *swarmRouter) getServiceLogs(ctx context.Context, w http.ResponseWriter
 	return nil
 }
 
+func (sr *swarmRouter) getTaskLogs(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+	if err := httputils.ParseForm(r); err != nil {
+		return err
+	}
+
+	// (dperny) i think it's safe to modify the vars/request like this but if
+	// there are REAL WEIRDO bugs later on down the line, y'all can double
+	// check here
+	r.Form.Add("tasks", vars["id"])
+	// TODO(dperny) i might should delete the id from vars before handling
+	// this request? but i don't think it matters
+	return sr.getSwarmLogs(ctx, w, r, vars)
```

Comment:
if we leave getSwarmLogs as is, with no `LogSelector` parameter, then later on a route for arbitrary combinations of service and task logs (for example, maybe stack logs?) can be easily constructed. in fact, on local, i had such a route, that looked somewhat like `/swarm/logs?service=foo&service=bar&task=wombo` and returned a log stream for whatever selectors.